pkg/lvmutils/utils.go

```diff
@@ -345,3 +341,12 @@ func ExtendThinpool(expansionTpSizePerBrick uint64, vgName string, tpName string
 	err := utils.ExecuteCommandRun("lvextend", fmt.Sprintf("-L+%dB", expansionTpSizePerBrick), fmt.Sprintf("/dev/%s/%s", vgName, tpName))
 	return err
 }
+
+// NormalizeSize converts the value to multiples of 512
```

Comment:
can we make it unexported function?

Reply:
this is used in `glusterd2/bricksplanner` as well